test/fixtures/assets/src/main.js

```diff
@@ -3,13 +3,13 @@ import App from 'fusion-core';
 import {assetUrl} from 'fusion-core';
 import * as jsonData from './static/test.json';
 
+import serverAsset from './server-asset.js';
+
 const hoistedUrl = assetUrl('./static/test.css');
 if (typeof window !== 'undefined') {
   window.__hoistedUrl__ = hoistedUrl;
 }
 
-import {serverAsset} from "./server-asset.js";
```

Comment:
Whoa, this file just didn't exist before? 😮

Reply:
It existed, but was empty 😞